pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -766,6 +766,37 @@ func TestRhpamTrialWithReposPersistedWithStorageClass(t *testing.T) {
 	assert.Len(t, env.Servers[0].PersistentVolumeClaims, 0)
 }
 
+func TestRhpamTrialWithoutSpecifyingReposPersistedWithStorageClass(t *testing.T) {
```

Comment:
why not reuse a existing test?
and where is the check to verify the persistRepos property?